src/components/Onboarding/Screens/ReviewInformation.js

```diff
@@ -19,4 +63,544 @@ function ReviewInformation({ title }) {
+        </div>
+        <p
+          css={`
+            ${textStyle('body4')};
+            color: ${theme.contentSecondary};
+            margin-bottom: ${3 * GU}px;
+          `}
+        >
+          If you don´t upload images, default ones will be displayed.
+        </p>
+      </div>
+
+      <LineBreak />
+      <div>
+        <div
+          css={`
+            margin-bottom: ${2 * GU}px;
+          `}
+        >
+          COMMUNITY LINKS
+        </div>
+        <div>
+          {Object.values(config.garden.links.community[0]).length > 0 ? (
+            config.garden.links.community.map(({ link, label }) => (
+              <Field label={label} value={link} />
+            ))
+          ) : (
+            <div
+              css={`
+                margin-bottom: ${3 * GU}px;
+              `}
+            >
+              No links
+            </div>
+          )}
+        </div>
+      </div>
+
+      <LineBreak />
+      <div>
+        <div
+          css={`
+            margin-bottom: ${2 * GU}px;
+          `}
+        >
+          DOCUMENTATION LINKS
+        </div>
+        <div>
+          {Object.values(config.garden.links.documentation[0]).length > 0 ? (
+            config.garden.links.documentation.map(({ link, label }) => (
+              <Field label={label} value={link} />
+            ))
+          ) : (
+            <div
+              css={`
+                margin-bottom: ${3 * GU}px;
+              `}
+            >
+              No links
+            </div>
+          )}
+        </div>
+      </div>
+    </div>
+  )
+}
+/// /////// TOKENOMICS //////////
+function ReviewGardenTokenomics() {
+  return (
+    <div
+      css={`
+        padding: ${5 * GU}px ${7 * GU}px;
+        width: 100%;
+      `}
+    >
+      <ReviewTokens />
+      <LineBreak />
+      <ReviewHoneyswapLiquidity />
+      <LineBreak />
+      <ReviewIssuance />
+    </div>
+  )
+}
+
+const ReviewTokens = () => {
+  const { config } = useOnboardingState()
+  // commonPool = ((totalSeedsAmount + gardenTokenLiquidity) * initialRatio) / (1 - initialRatio)
+  const totalSeedsAmount = config.tokens.holders.reduce(
+    (acc, [_, stake]) => acc + stake,
+    0
+  )
+  const initialRatio = config.issuance.initialRatio / 100
+
+  const commonPool =
+    parseFloat(
+      (totalSeedsAmount + parseInt(config.liquidity.tokenLiquidity)) *
+        initialRatio
+    ) / (1 - initialRatio).toFixed(2)
+
+  return (
+    <div>
+      <div
+        css={`
+          margin-bottom: ${2 * GU}px;
+        `}
+      >
+        TOKEN SETTINGS
+      </div>
+      <TwoCols>
+        {config.garden.type === BYOT_TYPE && (
+          <Field
+            label="Token address"
+            value={
+              <div>
+                {config.tokens.address}{' '}
+                <span>({config.tokens.existingTokenSymbol})</span>
+              </div>
+            }
+          />
+        )}
+        <Field label="Token name" value={config.tokens.name} />
```

Comment:
I notice that the Token address and name are super close without padding. What do you think of moving the Token address on the first row? And then the Token name and symbol on a second row.

Reply:
Do you think it could help to have the token address in an `IdentityBadge` ?

Reply:
I don't think is super relevant. Only to make it easy for the user to click the block explorer and check is the right one.